README.md

```diff
@@ -83,26 +83,36 @@ By default, you should start composing new scripts and test using forked network
 You have three forked networks to work with:
 
 - `mainnet-fork`
-- `goerli-fork`
 - `holesky-fork`
+- `sepolia-fork`
 
 To start new voting on the live networks you could proceed with:
 
 - `mainnet`
-- `goerli`
 - `holesky`
+- `sepolia`
 
-> Note: you can't run tests on the live networks.
+> [!CAUTION]
+> You can't run tests on the live networks.
 
 In a typical weekly omnibus workflow, you need only `mainnet-fork` and
 `mainnet` networks. In case of large test campaign on Lido upgrades,
-it also could be useful to go with `goerli` and `goerli-fork` testnets first.
+it also could be useful to go with `holesky` and `holesky-fork` testnets first.
 
+> [!IMPORTANT]
 > **Holesky specifics.**
 > Due to Holesky not being supported by Infura yet, setting RPC URL for Holesky is different. Instead of setting `WEB3_INFURA_PROJECT_ID` env variable set `HOLESKY_RPC_URL`.
+>
+> **Sepolia specifics.**
+> Due to Sepolia not being supported by Infura yet, setting RPC URL for Holesky is different. Instead of setting `WEB3_INFURA_PROJECT_ID` env variable set `SEPOLIA_RPC_URL`.
```

Comment:
see above

Reply:
Fixed in https://github.com/lidofinance/scripts/pull/208/commits/a5eb329c7dac82d23fceeb2a5d7927864f1e4426